guides/common/modules/proc_enrolling-project-server-in-freeipa-domain.adoc

```diff
@@ -0,0 +1,116 @@
+[id="enrolling-project-server-in-freeipa-domain_{context}"]
```

Comment:
@maximiliankolb Can you please take a look at the changes in this file and tell me if this would work for orcharhino? The idea is to use `.Procedure` to give only a high-level overview (because there are multiple ways to configure FreeIPA) with links to RHEL docs (included only for Satellite builds) and then give an example to show one of the possible roads to take (which I'm hoping would help make this module more user-friendly for non-Satellite builds too).

This makes things easier from the perspective of maintainability (as explained in https://github.com/theforeman/foreman-documentation/issues/3116#issuecomment-2220166876) but I would like to know if it breaks something for orcharhino. And if it does, I'd like to find a way to mitigate that.

Reply:
In general, links to RH docs are OK in the RHEL context, but not in the Satellite context. In downstream, I'd make them "RHEL"-only to hide them for all other OS. So this workflow is OK for me.

Works for me.

Reply:
@domiborges also suggested to include links to FreeIPA upstream resources (blogs, testing procedures) if we can find them. But for now, no change. If we do want to add them, let's do it in another PR then.